source/dll_main.cpp

```diff
@@ -905,6 +905,9 @@ BOOL APIENTRY DllMain(HMODULE hModule, DWORD fdwReason, LPVOID)
 		reshade::hooks::register_module(get_system_path() / L"opengl32.dll");
 		// Do not register Vulkan hooks, since Vulkan layering mechanism is used instead
 
+		reshade::hooks::register_module( L"openvr_api.dll" );
+		reshade::hooks::register_module( L"openvr_api64.dll" );
```

Comment:
This should probably be something like this to ensure only the DLL of the same bitness is hooked:
```cpp
#  ifdef WIN64
		reshade::hooks::register_module(L"openvr_api64.dll");
#  else
		reshade::hooks::register_module(L"openvr_api.dll");
#  endif
```

Reply:
I can definitely put the openvr_api64.dll behind a guard. However, for 64bit games the OpenVR dll is often just called `openvr_api.dll`, too, so we need to potentially hook this one in either case.